app_legacy/Helpers/database/game.php

```diff
@@ -248,7 +248,7 @@ function getGamesListByDev(
                            SUM(CASE WHEN ach.Author = :myRRDev THEN ach.TrueRatio ELSE 0 END) AS MyTrueRatio,
                            SUM(CASE WHEN ach.Author != :notMyAchDev THEN 1 ELSE 0 END) AS NotMyAchievements,
                            lbdi.MyLBs,";
-        $havingCond = "HAVING MyAchievements > 0 ";
+        $havingCond = "HAVING MyAchievements > 0 OR MyLBs > 0";
```

Comment:
The trailing space was necessary. I get an SQL error trying to test this without it.

Reply:
Fixed!